movie_api/settings.py

```diff
@@ -0,0 +1,129 @@
+"""
+Django settings for movie_api project.
+
+Generated by "django-admin startproject" using Django 5.1.3.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+import os
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / "subdir".
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.environ.get("SECRET_KEY")
+
+# SECURITY WARNING: don"t run with debug turned on in production!
+DEBUG = True
```

Comment:
The `DEBUG` setting is currently set to `True`. Remember to set it to `False` in a production environment to avoid exposing sensitive information.